packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -206,8 +208,8 @@ export class VirtualNode extends VirtualNodeBase {
           }
           : undefined,
         serviceDiscovery: {
```

Comment:
Shouldn't this render to `undefined` if `this. serviceDiscovery` is `undefined`?

Reply:
Good point. I think it should to match with the pattern we have elsewhere.